commands/view-state.js

```diff
@@ -36,8 +35,21 @@ async function viewState(options) {
     const { accountId, prefix, finality, blockId, utf8 } = options;
     const near = await connect(options);
     const account = await near.account(accountId);
-
-    let state = await account.viewState(prefix, { blockId, finality });
+    if (finality && blockId) {
```

Comment:
For mutually exclusive options (only one of either finality vs. blockId), we should try using .conflicts() (ref: https://yargs.js.org/docs/#api-reference-conflictsx-y)